extensions/email/src/main/java/org/mule/extension/email/internal/commands/ForwardCommand.java

```diff
@@ -37,18 +37,17 @@
      * The forwarded content is taken from the incoming {@link MuleMessage}'s payload. If not possible
      * this operation is going to fail.
      *
-     * @param connection     the connection associated to the operation.
-     * @param muleMessage    the incoming {@link MuleMessage} from which the email is going to getPropertiesInstance the content.
-     * @param content        the content of the email.
-     * @param subject        the subject of the email.
-     * @param from           the person who sends the email.
-     * @param defaultCharset the default charset of the email message to be used if the {@param content} don't specify it.
-     * @param toAddresses    the primary recipient addresses of the email.
-     * @param ccAddresses    the carbon copy recipient addresses of the email.
-     * @param bccAddresses   the blind carbon copy recipient addresses of the email.
+     * @param connection   the connection associated to the operation.
+     * @param email        the incoming {@link Email} from which the properties will be obtained.
+     * @param content      the content of the email.
+     * @param subject      the subject of the email.
+     * @param from         the person who sends the email.
+     * @param toAddresses  the primary recipient addresses of the email.
+     * @param ccAddresses  the carbon copy recipient addresses of the email.
+     * @param bccAddresses the blind carbon copy recipient addresses of the email.
      */
     public void forward(SenderConnection connection,
-                        MuleMessage muleMessage,
+                        Email email,
```

Comment:
`forward(email)` looks pretty declarative to me, but I do agree that a DisplayName is missing